essential-projects/core_contracts
# 🐛 fix IPublicQueryOptions-limit-type

## What did you change?

The value for limit does not only accept numbers, but also the string `'ALL'`. The interface `IPublicQueryOptions` did not reflect that. Now it does.

## How can others test the changes?

Try using `IPublicQueryOptions` and see how you can now use `limit: 'ALL'` without typescript shouting at you

## PR-Checklist

Please check the boxes in this list after submitting your PR:
